azext_iot/central/params.py

```diff
@@ -223,6 +223,50 @@ def load_central_arguments(self, _):
             help="The API version for the requested operation.",
         )
 
+    with self.argument_context("iot central device-group show") as context:
+        context.argument(
+            "device_group_id",
+            options_list=["--device_group_id", "-d"],
+            help="Unique ID of the device group."
+            "Regex pattern: ^[a-zA-Z0-9_-]*$",
+        )
+
+    with self.argument_context("iot central device-group create") as context:
+        context.argument(
+            "device_group_id",
+            options_list=["--device_group_id", "-d"],
+            help="Unique ID of the device group."
+            "Regex pattern: ^[a-zA-Z0-9_-]*$",
+        )
+        context.argument(
+            "content",
+            options_list=["--content", "-k"],
+            help="The device group definition. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example: ./path/to/file.json] [Example of stringified JSON: {<Device Group JSON>}]."
+        )
+
+    with self.argument_context("iot central device-group update") as context:
+        context.argument(
+            "device_group_id",
+            options_list=["--device_group_id", "-d"],
+            help="Unique ID of the device group."
+            "Regex pattern: ^[a-zA-Z0-9_-]*$",
+        )
+        context.argument(
+            "content",
+            options_list=["--content", "-k"],
+            help="The device group definition. Provide path to JSON file or raw stringified JSON."
```

Comment:
We have other commands that have similar inputs, like `az dt model create` specifically --models.

Try to follow those patterns to make the cli more unified. Also no need for the file path example and example of string json

Reply:
Sure.

Reply:
I followed the some patterns in iot central create commands here.